src/components/qr-home-button/index.tsx

```diff
@@ -0,0 +1,31 @@
+import Image from 'next/image';
+import React from 'react';
+
+interface QrHomeButtonProps {
```

Comment:
```suggestion
type QrHomeButtonProps = {
```
This is not wrong, but you must follow convention